docs/pages/introduction/why-not-expo.mdx

```diff
@@ -4,107 +4,49 @@ sidebar_title: Limitations
 ---
 
 import { InlineCode } from '~/components/base/code';
-import { Collapsible } from '~/ui/components/Collapsible';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BuildIcon, EasMetadataIcon } from '@expo/styleguide';
 
-Your success will be limited if you don't know the limitations of your tools. A good software engineer strives to understand the tradeoffs in the decisions they make.
+We believe Expo tools are a great option for anyone looking to create amazing cross-platform apps quickly. Nonetheless, everyone's project is unique, and there may be reasons Expo is not the right choice for what you're building. Read on for some reasons why you may **not** want to use Expo tools.
```

Comment:
```suggestion
We believe Expo tools are a great option for anyone looking to create amazing cross-platform apps quickly. Nonetheless, everyone's project is unique, and there may be reasons Expo is not the right choice for what you're building. Read on for some reasons why you may **not** want to use certain Expo tools with your project.
```
Just thinking of those who suggest Expo is a monolith, that you're either in or out, when really you can use just a little or a lot.